workspace/workspace_test.go

```diff
@@ -60,10 +60,11 @@ func TestWorkspaceExercises(t *testing.T) {
 	b2 := filepath.Join(tmpDir, "track-b", "exercise-two")
 
 	for _, path := range []string{a1, a2, b1, b2} {
+		path := filepath.Join(path, ignoreSubdir)
 		err := os.MkdirAll(path, os.FileMode(0755))
 		assert.NoError(t, err)
 
-		if path != a2 {
+		if path != filepath.Join(a2, ignoreSubdir) {
```

Comment:
> Tests having to know to create the `.exercism` ignore subdirectory seems smelly. I'm not sure how to avoid it though.

Reply:
I was thinking `os.MkdirAll` on the `filepath.Dir` of the full metadata file filepath  -- I'd rather do that than have to have the code know about the various pieces of the metadata file path.